core/templates/components/rubrics-editor/rubrics-editor.component.ts

```diff
@@ -95,6 +95,42 @@ export class RubricsEditorComponent {
     return Boolean(this.editableExplanations[difficulty][index]);
   }
 
+  ismediumLevelExplanationLengthEqualsOne(difficulty: string): boolean {
+    if (difficulty === this.skillDifficultyMedium) {
+      return this.rubrics[1]._explanations.length === 1;
+    }
+    return false;
+  }
+
+  isMediumLevelExplanationValid(difficulty: string, data: string): boolean {
```

Comment:
I want to create a test for this particular function, I can call `addExplanationForDifficulty` function to create an explanation but this function will initially create an empty string and it will result in fail for this test.
Thanks

Reply:
Does my general comment get around needing to test this?